hack/update-all.sh

```diff
@@ -90,4 +90,9 @@ do
 	fi
 done
 
-echo -e "${color_green}Update scripts completed successfully${color_norm}"
+if kube::util::ensure_clean_working_dir &>/dev/null; then
+    exec "${KUBE_ROOT}/hack/update-all-staging.sh" "$@"
```

Comment:
I don't agree, the amount of possibilities what can be updated is very different. I'd rather have a single pass if it's doable than being forced to run just because we didn't check if we can. This is dev tools, we're more forgivable, aren't we 😉